app/Phragile/AssigneeCollection.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Phragile;
+
+class AssigneeCollection {
+
+	private $assignees;
```

Comment:
Type hints for this field and for `$tasks` in the constructor are missing. Looks like they are both arrays of arrays, which is itself something to avoid. Still, indicating this with `array[]` is already more clear